api/opencl_runtime_layer.txt

```diff
@@ -7417,19 +7424,19 @@ argument does not return an event object.
 The execution status of an enqueued command at any given point in time can
 be one of the following:
 
-  * CL_QUEUED This indicates that the command has been enqueued in a
+  * {CL_QUEUED} This indicates that the command has been enqueued in a
```

Comment:
These should probably be the anchors for these values (they currently have no anchors).

Reply:
This comment applies to this entire list.

Reply:
Fixed - thanks!